Sources/NIOCore/NIOLoopBound.swift

```diff
@@ -17,7 +17,7 @@
 ///
 /// ``NIOLoopBound`` is useful to transport a value of a non-`Sendable` type that needs to go from one place in
 /// your code to another where you (but not the compiler) know is on one and the same ``EventLoop``. Usually this
-/// involves `@Sendable` closures. This type is safe because it verifies (using ``EventLoop/preconditionInEventLoop(file:line:)-2fxvb``)
+/// involves `@Sendable` or `sending` closures. This type is safe because it verifies (using ``EventLoop/preconditionInEventLoop(file:line:)-2fxvb``)
```

Comment:
Unrelated, but thought this would be more precise.

Reply:
Related: should I add more explicit documentation outlining that there are a bunch of functions to make a `NIOLoopBoundBox` off the event loop called `make*`, and outline the use cases for them at a high level? Separate PR?